packages/admin-app/src/components/AssignmentTable/AssignmentTable.tsx

```diff
@@ -56,7 +58,7 @@ export const AssignmentTable: React.FC<AssignmentTableProps> = ({
                         }}
                         hideFooterPagination
                         selectionModel={selectionModel}
-                        checkboxSelection
+                        checkboxSelection={!!onChanged}
```

Comment:
In other words AssignmentTable was previously for selecting users, but now it has been repurposed to just display users